strfreq.c

```diff
@@ -0,0 +1,32 @@
+/*Rumeet Goradia - String Character Frequency*/
+#include <stdio.h>
```

Comment:
Again, more comments! But good job! There was a weird thing with an extra = 1 being printing out, but you handled the different test cases correctly!